Digipost.Api.Client.DataTypes/Payslip.cs

```diff
@@ -0,0 +1,7 @@
+namespace Digipost.Api.Client.DataTypes
+{
+    public class Payslip
```

Comment:
Denne klassen er helt tom. Mangler det noe her?

Reply:
Den klassen eksisterer også i DataTypes repoet, så jeg lagde en fil for den her. Fant ut senere at den er tom i Datatypes også. Vi kan vel fjerne både her og der?

Reply:
Fikset i 8c23d67

Reply:
Men Payslip *er jo* en datatype, så den må jo være her, eller?